src/server/pfs/server/driver_file.go

```diff
@@ -490,3 +490,12 @@ func (d *driver) getOrComputeTotal(ctx context.Context, commit *pfs.Commit) (*fi
 	}
 	return d.commitStore.GetTotalFileset(ctx, commit)
 }
+
+// sizeOfCommit gets the size of a commit.
+func (d *driver) sizeOfCommit(ctx context.Context, commit *pfs.Commit) (int64, error) {
+	fsid, err := d.getFileset(ctx, commit)
+	if err != nil {
+		return 0, err
+	}
+	return d.storage.SizeOf(ctx, *fsid)
```

Comment:
Just to make sure we are on the same page, this might not be the actual size of the commit since deletions may not have been applied yet. I think we need to actually do a merge read to compute this.

Reply:
Yeah.  I think we can make that change in SizeOf.  Maybe we should have a method that returns a range?  And we can just use the sizes of the additive and deletive set?  Then you can get approximate sizes without too much work.